docs/how-to-guides/integrating-autoware/creating-vehicle-and-sensor-description/creating-vehicle-description.md

```diff
@@ -0,0 +1,205 @@
+# Creating vehicle description
```

Comment:
This title doesn't cover all of the content. This page includes vehicle_description and vehicle_launch packages.

```suggestion
# Creating vehicle model for Autoware
```